### PR TITLE
Minor fixes + column formatter feature

### PR DESCRIPTION
**Pull request by Wieltje**
Would be great if you could pull these changes and create a new release. If you have any comments or questions just let me know.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -319,7 +319,7 @@ export function getLogger(context) {
  */
 export function setLogLevel(logLevel, logger) {
 
-    if (logLevel instanceof Number) {
+    if (Number.isInteger(logLevel)) {
```
Comment (anigenero) [inline src/index.js]: Modifying this separately. Good catch on defect.

File: src/formatter.js [hunk 2]
```diff
@@ -452,11 +471,14 @@ let _getFileDetails = function (logEvent) {
...
-
+		
 		if (typeof define !== 'undefined') {
 			let path = require('path');
 			let appDir = path.dirname(require.main.filename);
-			logEvent.filename = fileParts.join(':').replace(appDir, '').replace(/(\\|\/)/, '');
+			if (!fileParts[0].startsWith(appDir)) {
+				appDir = '';
+			}
+			logEvent.filename = fileParts.join(':').replace(appDir, '').replace(/^(\\|\/)/, '');
```
Comment (anigenero) [inline src/formatter.js]: What's the reasoning behind this? Are you trying to resolve absolute paths?

File: src/formatter.js [hunk 3]
```diff
@@ -230,6 +248,7 @@ let _formatters = {
 	'F|file' : _formatFile,
 	'K|map|MAP' : _formatMapMessage,
 	'L|line' : _formatLineNumber,
+	'column': _formatColumn,
```
Comment (anigenero) [inline src/formatter.js]: This is not a layout pattern specified in the log4j2 specification (which is what I'm trying to replicate), but I'm actually fine with it.

File: src/formatter.js (hunk 2 above)
Comment (Wieltje) [inline src/formatter.js]: I got situations where appDir was "." or "/" while that wasn't the beginning of fileParts[0], resulting in the first "/" or "." later on in the string being removed. This solution should prevent this.

File: src/formatter.js (hunk 3 above)
Comment (Wieltje) [inline src/formatter.js]: Great! When using as replacement of console.log, this was one of the things I was missing.

**Comment by anigenero** [comment]
All changes have been released in 1.0.2

**Comment by Wieltje** [comment]
Thanks for your effort!

**Pull request merged**